docker-compose.yml

```diff
@@ -10,25 +10,3 @@ services:
     ipc: host
     ports:
       - "3001:80"
-
```

Comment:
I wouldn't remove these from the docker-compose while this can be handy for local testing/solving containerization problems. No caching should be used though.

Reply:
We're running tests as well as the linter directly on the command line. The settings in `docker-compose` have only been used by Jenkins. In which situations do you see a use for these config settings here?